### DISCUSSION TITLE
Feature request: add expected end time of charging

### DISCUSSION THREAD

**Discussion by rtega** [teslamate-org/teslamate#1567] [Ideas]
It would be nice if it would be possible to show the expected end time of charging on the summary page of teslamate. This would be presumably after or before "Remaining Time".

Something like:

```
Remaining time    4h 15 min
Expected end time 16:15

```

**Comment by adriankumpf** [comment]
Feature suggestions belong here: [💡 Ideas](https://github.com/adriankumpf/teslamate/discussions/categories/ideas). I'll move it.

**Reply by rtega** [reply]
Ah, didn't know. Thanks!
